### PR TITLE
add attmap recipe

### PR DESCRIPTION
**Pull request by michalstolarczyk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/attmap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - pip
+    - python
+  run:
+    - python
+    - ubiquerg >=0.2.1
+
+test:
+  imports:
+    - attmap
+  requires:
```
Comment (xhochy) [inline recipes/attmap/meta.yaml]: These should be removed, not needed for import tests.

**Comment by michalstolarczyk** [comment]
@xhochy is this ready? builds succeeded

File: recipes/attmap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 95b1f7dbcdad7278a3702fa921be6271046c96e1c9ed9feb10e0d4c13092b0a0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/attmap/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/attmap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 95b1f7dbcdad7278a3702fa921be6271046c96e1c9ed9feb10e0d4c13092b0a0
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/attmap/meta.yaml]:
```suggestion
  number: 0
  noarch: python
```

**Pull request merged**